src/seedu/addressbook/AddressBook.java

```diff
@@ -383,12 +394,88 @@ private static String executeCommand(String userInputString) {
             return getUsageInfoForAllCommands();
         case COMMAND_EXIT_WORD:
             executeExitProgramRequest();
+            //Exits program, no return expected.
+        case COMMAND_SORT_WORD:
+        	return executeSortAndListAllPersonsInAddressBook();
+        case COMMAND_EDIT_WORD:
+        	return executeEditPerson(commandArgs);
         default:
             return getMessageForInvalidCommandInput(commandType, getUsageInfoForAllCommands());
         }
     }
 
     /**
+     * Edit a person based on index shown in last list/find.
+     * @param commandArgs
+     * @return
+     */
```

Comment:
Header comments start with `Edits` instead of `edit` because it reads as `This method edits ...`. You also need to give description to return and param.